README.md

```diff
@@ -44,4 +44,12 @@ For example replace `SRC_REPO_TOKEN` with `github-user:github-access-token`
 | `IGNORE_REFS_PATTERN` | Don't mirror matching refs | No, default: `refs/pull` | `refs/pull` |
 | `TLS_TOFU` | Enable / Disable [TLS-Trust On First Use] | No, default: `false` | `true` or `false` |
 
+Ignoring multiple refs is possible by separating them with space, but you also need to set MIRROR to false otherwise, sync will delete your branches. For example:
```

Comment:
I do think you make a valid point here. This made me actually thinking if we should not just drop the MIRROR option entirely and never use `git push --mirror` . Synchronizing deleted branches has to be implemented in a different way anyways...

Reply:
Yes, I would put MIRROR to be false by default. In two-way sync, it is not possible to delete branch in any repo, because on the next sync, the branches you deleted will be recreated again. This happens because they were never deleted from temporary repo withing docker container.

So, if MIRROR is on, you need to stop the sync, delete branches and then restart sync.

Reply:
Agree, will take care of this in a different pull request...